src/ucs/datastruct/ptr_map.inl

```diff
@@ -105,19 +116,25 @@ ucs_ptr_map_get(const ucs_ptr_map_t *map, ucs_ptr_map_key_t key)
  *
  * @param [in]  map     Container.
  * @param [in]  key     Key to object pointer.
+ * @return      UCS_OK on success otherwise error code as defined by
+ *              @ref ucs_status_t.
```

Comment:
UCS_OK on success
UCS_ERR_NO_ELEM if the key is not found in the internal hash table